### PR TITLE
fix(settings): fix settings sub routes

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Fix param capturing for settings sub routes.

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to settings.
- Check that you can open the edit form for users, license keys, DHCP snippets and package repos.
- Check that other settings routes function as they did before.

## Fixes

Fixes: #4183.

### CODE REVIEW

File: docs/HACKING.md [hunk 1]
```diff
@@ -497,6 +497,12 @@ Now create a fake Windows image:
 dd if=/dev/zero of=windows-dd bs=512 count=10000
 ```
 
+Move the image somewhere that the snap can access:
+
+```shell
+mv windows-dd /var/snap/maas/common/windows-dd
+```
+
```
Comment (petermakowski) [inline docs/HACKING.md:505]: Hmm, this would probably need some QA. Should this go to a separate PR?

File: docs/HACKING.md [hunk 2]
```diff
@@ -497,6 +497,12 @@ Now create a fake Windows image:
 dd if=/dev/zero of=windows-dd bs=512 count=10000
 ```
 
+Move the image somewhere that the snap can access:
+
+```shell
+mv windows-dd /var/snap/maas/common/windows-dd
```
Comment (petermakowski) [inline docs/HACKING.md:503]: This fails with permission denied if using multipass. We should either update the multipass command to `multipass exec dev-maas -- sudo bash` or add sudo to this one.

```suggestion
sudo mv windows-dd /var/snap/maas/common/windows-dd
```

**Review by petermakowski — APPROVED** [review]

File: docs/HACKING.md (hunk 1 above)
Comment (huwshimi) [reply docs/HACKING.md:505]: Sure, I've created an issue: https://github.com/canonical-web-and-design/maas-ui/issues/4207 and will remove these changes from this PR.
